csharp/ql/test/query-tests/Nullness/NullMaybeBad.cs

```diff
@@ -1,18 +1,15 @@
 using System;
 
-namespace NullMaybe
+class Bad
```

Comment:
I think we actually want to sync the other way around, i.e., copy the files from the tests to the samples. That way the test files for NullMaybe and NullAlways will follow the same structure (having a namespace).

Reply:
All of the samples I've been browsing don't have a namespace, which I would agree with because it's good to keep them as short as possible.